test/functional/test_framework/util.py

```diff
@@ -57,6 +57,11 @@ def assert_equal(thing1, thing2, *args):
         raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
 
 
+def assert_not_equal(thing1, thing2, *args):
```

Comment:
2103e12: maybe just have `assert_not_equal(thing1, thing2)`. i couldn't find an occurrence where we needed to `assert_not_equal` 3+ things. ex: `assert_not_equal(2, 2, 3)`

Reply:
i think you can find more places where `assert_not_equal` can be used using the command @theStack mentioned [here](https://github.com/bitcoin/bitcoin/issues/23119#issue-1009729239)!

example first 3 lines output:
```
$ git grep "assert ".*\!=
feature_fee_estimation.py:        assert fee_dat_current_content != fee_dat_initial_content
feature_fee_estimation.py:        assert fee_dat_current_content != fee_dat_initial_content
mempool_accept_wtxid.py:        assert child_one_wtxid != child_two_wtxid
```